### PR TITLE
Update to latest osu

### PR DESCRIPTION
**Pull request by Shawdooow**

### CODE REVIEW

File: osu.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.26228.9
```
Comment (ColdVolcano) [inline osu.sln]: rip vs 2015

Comment (Jorolf) [inline osu.sln]: it's for the better

Comment (Jorolf) [inline osu.sln]: let it die

Comment (Jorolf) [inline osu.sln]: his accomplishments will be remebered

**Pull request merged**